### ISSUE TITLE
Support multi-class and multi-output classification

### ISSUE THREAD

**Issue by robertavram-md** [HeartWise-AI/Orion#21]
### Description
Currently, the code trains video models for a single outcome (`Outcome`) per sample. However, with our recent dataset from Labelbox, the `Outcome` now contains tuples representing multiple attributes (e.g., `(5,1,1)` indicating a right coronary with stenting and contrast).

**Tasks to Complete:**
1. **Modify Data Processing:**
   - Adapt data loading to handle `Outcome` columns containing tuples.
   - **Example:**
     ```csv
     FileName,Outcome,Target_loc_label
     video1.avi,"(5,1,1)",5
     video2.avi,"(9,0,1)",9
     ```

2. **Adjust the Loss Function:**
   - Implement a loss function suitable for multi-output, such as `MultiLabelSoftMarginLoss` or a combination of individual losses for each attribute.

3. **Update the Neural Network’s Classification Head:**
   - Modify the output layer to predict multiple labels simultaneously.
   - **Example:** For three attributes, the output layer should have three neurons.

4. **Update Training and Evaluation Processes:**
   - Ensure metrics account for multiple outputs.
   - **Example:** Calculate accuracy for each attribute individually and overall.

**Examples:**

- **Current Data Format:**
  ```csv
  FileName,Outcome,Target_loc_label
  video1.avi,5,Right_coronary
  video2.avi,9,Left_coronayr
  
  -**New Data Format**
  FileName,Outcome,Target_loc_label
video1.avi,"(5,1,1)",(right_coronary, stenting, contrast)
video2.avi,"(9,0,1)",(left_coronary, stenting, contrast)
  
-**Suggested changes**
-Change video.py to support Outocme as a tuple  as well as Target_loc_label as a tuple
- Support a new loss `loss: multi_label_loss` in config.yaml and train.py
- Make it so that `num_classes ` can be a tuple (3,1,1) for 3 classes for outcome in position 0, 1 class in position 1 and 1 class in position 2
- Change neural network head
-
